README.md

```diff
@@ -530,6 +536,18 @@ startupProbe:
 We don't have a healthcheck endpoint in our app, so lets add one that just returns 200.
 We can then publish our app, along with our helm chart, and watch what happens.
 
+### Persistent Volumes
+
+The app stores the output images in a local folder.
+If an image is processed by a different Pod to the one receiving the request then the output image will not be visible.
+
+We can resolve this by attaching the same Persistent Volume to all our Pod instances.
+Configure a Persistent Volume using the `azurefiles` storage type in your Helm Chart and add a Persistent Volume Claim to your Deployment definition.
```

Comment:
I haven't had a chance to try this yet, but it seems like an interesting stretch goal